src/build/build_options.c

```diff
@@ -250,6 +250,16 @@ void append_file(BuildOptions *build_options)
 	vec_add(build_options->files, current_arg);
 }
 
+void append_arg(BuildOptions *build_options)
+{
+	if (vec_size(build_options->args) == MAX_FILES)
```

Comment:
A copy-paste typo from `append_file`

```suggestion
	if (vec_size(build_options->args) == MAX_ARGS)
```